See config/dkrz/levante_intel_patches/README.md

--- a/rrtmgp/kernels/mo_gas_optics_kernels.F90
+++ b/rrtmgp/kernels/mo_gas_optics_kernels.F90
@@ -51,7 +51,7 @@ contains
     integer,     dimension(ncol,nlay), intent(out) :: jtemp, jpress
     logical(wl), dimension(ncol,nlay), intent(out) :: tropo
     integer,     dimension(2,    ncol,nlay,nflav), intent(out) :: jeta
-    real(wp),    dimension(2,    ncol,nlay,nflav), intent(out) :: col_mix
+    real(wp),    dimension(:,       :,   :,    :), intent(out) :: col_mix
     real(wp),    dimension(2,2,2,ncol,nlay,nflav), intent(out) :: fmajor
     real(wp),    dimension(2,2,  ncol,nlay,nflav), intent(out) :: fminor
     ! -----------------
